src/components/navigation/Accordion.tsx

```diff
@@ -58,7 +53,12 @@ const AccordionItem = React.forwardRef<HTMLDivElement, AccordionItemProps>(funct
 			padding={{ all: 'small' }}
 			height={40}
 			ref={ref}
-			style={{ minWidth: 0, flexBasis: 0, flexGrow: 1 }}
+			style={{
+				minWidth: 0,
+				flexBasis: 0,
+				flexGrow: 1,
+				maxWidth: item.items && item.items.length > 0 ? 'calc(100% - 32px)' : '100%'
```

Comment:
I'm not sure this is required, can you provide an example where accordion is overflowing without this rule?